nbdev/quarto.py

```diff
@@ -147,8 +147,10 @@ def nbdev_readme(
     _render_readme(cfg_path, path, chk_time)
     if (tmp_doc_path/'README.md').exists():
         _rdm = cfg_path/'README.md'
+        _rdmi = tmp_doc_path/(Path(get_config().readme_nb).stem + '_files')
```

Comment:
You can remove Path()  here because '/' can take str as rhs

Reply:
This value that is returned isn't a path though

Reply:
I think it clicked with me just now that the LHS `tmp_doc_path` is a path, even though `get_config().readme_nb` is not!  

Sorry about that.  I can change this

Reply:
Actually this doesn't work.  I actually just want the stem of the RHS only, just ignore my previous message :)

Reply:
Oh right I get it now. I didn't read your original carefully enough